compiler/src/iree/compiler/Dialect/Flow/Transforms/DumpDispatchGraph.cpp

```diff
@@ -138,9 +138,11 @@ class DumpDispatchGraphPass
     auto modOp = dyn_cast<ModuleOp>(getOperation());
     if (!modOp) return;
 
-    for (auto funcOp : modOp.getOps<func::FuncOp>()) {
+    auto funcOps = modOp.getOps<func::FuncOp>();
+
+    if (!funcOps.empty()) {
```

Comment:
CHange this to
```
if (funcOps.empty()) continue;

emitGraph(...);
```

Reply:
Thanks. I made it return if it is empty.